### PR TITLE
Add header fixing

### PR DESCRIPTION
**Pull request by lucasmichot**
For StyleCI, this can be added via the web interface.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
NB, @briannesbitt, you can get StyleCI to do this too from the setting page on the web interface.

File: src/Carbon/Lang/de.php [hunk 1]
```diff
@@ -3,11 +3,10 @@
 /*
  * This file is part of the Carbon package.
  *
- * (c) Michael Hohl <me@michaelhohl.net>
```
Comment (OskarStark) [inline src/Carbon/Lang/de.php]: is this correct?

Comment (lucasmichot) [inline src/Carbon/Lang/de.php]: Yes. If there are support or issues, people shall contact @briannesbitt and not the committer of this files or change.

Comment (GrahamCampbell) [inline src/Carbon/Lang/de.php]: > is this correct?

Yes, all files are copyright Brian, even if he didn't author them.

Comment (OskarStark) [inline src/Carbon/Lang/de.php]: :+1: thanks for the feedback

**Comment by briannesbitt** [comment]
This duplicates the header in the test files... 
https://github.com/lucasmichot/Carbon/blob/feature/master.phpcs-header/tests/Carbon/AddTest.php

**Comment by lucasmichot** [comment]
Fixed @briannesbitt 
(Namespaces decarations had to be moved under the existing file docblock)

**Comment by briannesbitt** [comment]
k cool

**Pull request merged**